sources/packages/api/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts

```diff
@@ -36,6 +36,14 @@ export class DisbursementScheduleErrorsService extends RecordDataModelService<Di
       newDisbursementsFeedbackErrors.dateReceived = dateReceived;
       return newDisbursementsFeedbackErrors;
     });
-    return this.repo.save(errorCodesObject);
+    return this.repo
+      .createQueryBuilder()
+      .insert()
+      .into(DisbursementFeedbackErrors)
+      .values(errorCodesObject)
+      .onConflict(
+        `ON CONSTRAINT disbursement_schedule_id_error_code_unique DO NOTHING`,
```

Comment:
Don't need the interpolation back ticks if nothing is being interpolated 😉

Reply:
updated